src/moaicore/MOAICCParticleSystem.h

```diff
@@ -188,34 +211,72 @@ class MOAIParticlePexPlugin :
 	static int		_setTangentalAcceleration			( lua_State* L );
 	static int		_getTangentalAccelerationVariance	( lua_State* L );
 	static int		_setTangentalAccelerationVariance	( lua_State* L );
-
+	
+	static int		_getGravityVariance					( lua_State* L );
+	static int		_setGravityVariance					( lua_State* L );
+	static int		_getRotationalAcceleration			( lua_State* L );
+	static int		_setRotationalAcceleration			( lua_State* L );
+	static int		_getRotationalAccelerationVariance	( lua_State* L );
+	static int		_setRotationalAccelerationVariance	( lua_State* L );
+	
 	static int		_getTextureName						( lua_State* L );
+	
+	static int		_getTotalParticles					( lua_State* L );
+	static int		_setTotalParticles					( lua_State* L );
+	static int		_getParticleCount					( lua_State* L );
+	static int		_setEmissionRate					( lua_State* L );
+	static int		_getParticlePositionType			( lua_State* L );
+	static int		_setParticlePositionType			( lua_State* L );
+	
+	static int		_getIndexMode						( lua_State* L );
+	static int		_setIndexMode						( lua_State* L );
+	static int		_getStartIndex						( lua_State* L );
+	static int		_setStartIndex						( lua_State* L );
+	static int		_getEndIndex						( lua_State* L );
+	static int		_setEndIndex						( lua_State* L );
+	static int		_getDeckTileCount					( lua_State* L );
+	static int		_getDeckTileDimensions				( lua_State* L );
+	static int		_getDeckCellDimensions				( lua_State* L );
+	static int		_getDeckTileOffset					( lua_State* L );
+	
+	static int		_flipY								( lua_State* L );
 	static int		_initializeProperties				( lua_State* L );
 	static int		_load								( lua_State* L );
-
-	static void     InitializeBasicRegisters    ( MOAIParticlePexPlugin& plugin );
-	static void		InitializeEmitter			( MOAIParticlePexPlugin& plugin );
-	void			InitializeProperties		();
-	static void		Parse						( cc8* filename, MOAIParticlePexPlugin& plugin, TiXmlNode* node );
-	
-	void			_initGravityScript			( float* particle, float* registers );
-	void			_initRadialScript			( float* particle, float* registers );
-	void			_renderGravityScript		( float* particle, float* registers, AKUParticleSprite* sprite, float t0, float t1,  float term );
-	void			_renderRadialScript		    ( float* particle, float* registers, AKUParticleSprite* sprite, float t0, float t1,  float term );
-
+	static int		_startSystem						( lua_State* L );
+	static int		_stopSystem       					( lua_State* L );
+	static int      _surge								( lua_State* L );
```

Comment:
Could you make sure the spacing you commit matches the spacing the rest of Moai is using?